includes/validation/class-amp-validated-url-post-type.php

```diff
@@ -615,7 +615,7 @@ public static function get_url_from_post( $post ) {
 		$url = $post->post_title;
 
 		$queried_object = get_post_meta( $post->ID, '_amp_queried_object', true );
-		$is_amp_story   = isset( $queried_object['id'], $queried_object['type'] ) && 'post' === $queried_object['type'] && AMP_Story_Legacy_Post_Type::POST_TYPE_SLUG === get_post_type( $queried_object['id'] );
+		$is_amp_story   = isset( $queried_object['id'], $queried_object['type'] ) && 'post' === $queried_object['type'] && ( AMP_Story_Legacy_Post_Type::POST_TYPE_SLUG === get_post_type( $queried_object['id'] || AMP_Story_Post_Type::POST_TYPE_SLUG === get_post_type( $queried_object['id'] ) ) );
```

Comment:
```suggestion
		$is_amp_story   = isset( $queried_object['id'], $queried_object['type'] ) && 'post' === $queried_object['type'] && ( in_array( [ AMP_Story_Legacy_Post_Type::POST_TYPE_SLUG, AMP_Story_Post_Type::POST_TYPE_SLUG ], get_post_type( $queried_object['id'] ), true ) );
```

Reply:
This code is cleaner and fixes an issue with `get_post_type( $queried_object['id']` that doesn't have a closing bracket.

Reply:
Updated within 178f07b